src/iop/globaltonemap.c

```diff
@@ -695,9 +695,9 @@ void gui_init(struct dt_iop_module_t *self)
   g->operator= dt_bauhaus_combobox_new(self);
   dt_bauhaus_widget_set_label(g->operator, NULL, _("operator"));
 
-  dt_bauhaus_combobox_add(g->operator, "reinhard");
-  dt_bauhaus_combobox_add(g->operator, "filmic");
-  dt_bauhaus_combobox_add(g->operator, "drago");
+  dt_bauhaus_combobox_add(g->operator, C_("global tonemap operator", "reinhard"));
```

Comment:
Here the context is "options" or "choice".

Reply:
I only added context to these so as to avoid having the same string for the "filmic" module name and the tonemap operator of the same name - but again, come to think of it, might be better to add "modulename" context to the other "filmic" instead :)